fuzz/generate-files.sh

```diff
@@ -40,13 +41,13 @@ done
 cat > "$REPO_DIR/.github/workflows/fuzz.yml" <<EOF
 # Automatically generated by fuzz/generate-files.sh
 name: Fuzz
-
 on:
-  push:
-    branches:
-      - master
-      - 'test-ci/**'
-  pull_request:
+  schedule:
+    # 5am every day UTC, this correlates to:
+    # - 11pm PDT
+    # - 7am CET
+    # - 5pm AEDT
+    - cron: '00 06 * * *'
```

Comment:
This is 6AM right?

Reply:
We need to change this to `05` or update the comment above.
I know that this is "copy-pasta" left over from the another PR that I saw earlier that had both `06` and `6am`

Reply:
Yep. (At least, that's how I read it.)

Reply:
Oh woops, I changed the three bottom ones and missed the top one (not shown here). While investigating I noticed that I totally botched this PR, I didn't change the output file to `cron-fuzz-daily.yml` and I didn't run the script either - god damn, it is hard to get good help. Will follow up.

Reply:
Man, now I'm confused. A job cron-daily-fuzz ran last night.

Reply:
Oh, I must have created the yaml file manually and not run `generate-files.sh` - face palm.

Reply:
https://github.com/rust-bitcoin/rust-miniscript/pull/684

Thanks for the careful review @storopoli!

Reply:
I'm also confused -- I ran the script locally and the diff was clean (didn't even create any other files). Are you saying it did nothing?

Reply:
Seems its too early in the morning for my brain to context switch back into this but if you check `cron-daily-fuzz.sh` you will ask yourself "who wrote this rubbish and how did it get in master?" For example it still runs on `pull_request` - all fixed in #684. As for postmortem I'm not sure how I botched it so badly.

Reply:
I think I must have tested the fuzz script in a different repo or something. This is definitely broken as hell :).